tpl/bootstrap.php

```diff
@@ -159,7 +159,16 @@
 ?>
 					<li>
 						<div id="password" class="navbar-form hidden">
-							<input type="password" id="passwordinput" placeholder="<?php echo I18n::_('Password (recommended)'); ?>" class="form-control" size="19" />
+
+							<!-- The following form element glues pseudo-login data together, so that password managers can try to look up stored value pairs.
+							     The username field is made hidden, because I found that otherwise password managers will fill in data into the password field.
```

Comment:
As nobody knows who is `I` when this code is merged, you might either express this in passive or just remove the part and say that password managers (may) do so. The fact that someone found this is less important.
Instead of "make hidden" just say it "is hidden".